psi4/src/psi4/libqt/qt.h

```diff
@@ -140,7 +140,7 @@ void C_DTRMV(char uplo, char trans, char diag, int n, double* a, int lda, double
 void C_DTRSM(char side, char uplo, char transa, char diag, int m, int n, double alpha, double* a, int lda, double* b, int ldb);
 
 // BLAS 3 Double routines
-void C_DGEMM(char transa, char transb, int m, int n, int k, double alpha, double* a, int lda, double* b, int ldb, double beta, double* c, int ldc);
+PSI_API void C_DGEMM(char transa, char transb, int m, int n, int k, double alpha, double* a, int lda, double* b, int ldb, double beta, double* c, int ldc);
```

Comment:
is there a reason why only `DGEMM` is being exported?

Reply:
The current exports are exactly enough to get the packaged plugin templates to compile. I figure we need an en banc judgement on what exactly is the public c-side psi api.